doc/releases/release-notes-3.6.rst

```diff
@@ -138,6 +138,9 @@ Build system and infrastructure
 * Added MCUboot image size reduction to sysbuild images which include MCUboot which prevents
   issues with building firmware images that are too large for MCUboot to swap.
 
+* Fixed an issue whereby shields were processed in order of the root they resided in rather than
+  the order there were supplied to cmake in.
```

Comment:
Typo + incomplete sentence (or extra "in")?

```suggestion
  the order they were supplied to cmake in xxx?
```

Reply:
Reads fine to me "they were supplied to cmake in" whereby in being the input

Reply:
ah! of course :) 
Thanks!